MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs

```diff
@@ -429,49 +426,51 @@ private async Task ConvertTextSubtitleToSrtInternal(string inputPath, string lan
                 encodingParam = " -sub_charenc " + encodingParam;
             }
 
-            var process = _processFactory.Create(new ProcessOptions
+            var processStartInfo = new ProcessStartInfo
             {
                 CreateNoWindow = true,
                 UseShellExecute = false,
                 FileName = _mediaEncoder.EncoderPath,
                 Arguments = string.Format("{0} -i \"{1}\" -c:s srt \"{2}\"", encodingParam, inputPath, outputPath),
-                EnableRaisingEvents = true,
-                IsHidden = true,
+                WindowStyle = ProcessWindowStyle.Hidden,
                 ErrorDialog = false
-            });
+            };
 
-            _logger.LogInformation("{0} {1}", process.StartInfo.FileName, process.StartInfo.Arguments);
+            int exitCode;
 
-            try
-            {
-                process.Start();
-            }
-            catch (Exception ex)
+            using (var process = new Process { StartInfo = processStartInfo, EnableRaisingEvents = true })
```

Comment:
Note to reviewers: this diff is significantly easier to read if you turn off white-space differences